src/App.jsx

```diff
@@ -1,33 +1,46 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
+import { Header } from './components/Header';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
-          </div>
-        </div>
-      </div>
+function prepareString(string) {
+  return string.toLowerCase().trim();
+}
 
-      <MoviesList movies={moviesFromServer} />
-    </div>
+function getPreparedMovies(movies, { query }) {
+  let preparedMovies = [...movies];
+
+  if (query) {
+    preparedMovies = preparedMovies.filter(
+      movie => prepareString(movie.title).includes(prepareString(query))
+        || prepareString(movie.description).includes(prepareString(query)),
+    );
+  }
+
+  return preparedMovies;
+}
```

Comment:
This code looks good, but if you invert the check you will make it better

Also you can use destructuring for `movie` 
You call `prepareString(query)` twice, let's create a separate variable for it.

```suggestion
function getFilteredMovies(movies, { query }) {
  if (!query) {
    return movies;
  }

  return movies.filter(
    movie =>
      prepareString(movie.title).includes(prepareString(query)) ||
      prepareString(movie.description).includes(prepareString(query)),
  );
}
```